TestFoundation/TestCodable.swift

```diff
@@ -254,19 +258,31 @@ class TestCodable : XCTestCase {
     // MARK: - CGRect
     lazy var cgrectValues: [CGRect] = [
         CGRect(),
+        CGRect.zero,
         CGRect(origin: CGPoint(x: 10, y: 20), size: CGSize(width: 30, height: 40)),
         CGRect(origin: CGPoint(x: -10, y: -20), size: CGSize(width: -30, height: -40)),
         // Disabled due to limit on magnitude in JSON. See SR-5346
-        // CGRect(origin: CGPoint(x: -.greatestFiniteMagnitude / 2,
-        //                        y: -.greatestFiniteMagnitude / 2),
-        //        size: CGSize(width: .greatestFiniteMagnitude,
-        //                     height: .greatestFiniteMagnitude)),
+        // CGRect.infinite
     ]
     
     func test_CGRect_JSON() {
         for rect in cgrectValues {
             expectRoundTripEqualityThroughJSON(for: rect)
         }
+        
+        do {
```

Comment:
See my comment above about this - I'm just thinking from a future maintenance point of view.